src/subcommands/deploy.rs

```diff
@@ -51,6 +51,11 @@ pub struct Deploy {
     )]
     estimate_only: bool,
     #[clap(long, help = "Wait for the transaction to confirm")]
+    salt: Option<String>,
+    #[clap(
+        long,
+        help = "Use the given salt to pre-compute contract deploy address"
```

Comment:
The help text here seems a bit confusing to me. It sounds like the salt is _only_ used to pre-compute the address but not for actual deployment.

Reply:
That's totally true I don't know why I kept the "pre".